content/en/apps/features/integrations/dhis2.md

```diff
@@ -51,20 +51,20 @@ CHWs support patients in their community by following care guides and recording
 
 ### Supervisor
 
-In addition to their own Targets, Supervisors can see the aggregate of each DHIS2 Data Value across *all* CHWs in their area from the CHW Aggregate view on the Targets tab. By tapping on a Target, they can also see each CHW’s contribution towards that total. Supervisors can review and verify with CHWs that everyone has synced and their data is correct. The Supervisor can communicate with the HRIO when everything has been validated.
+In addition to their own targets, Supervisors can see the aggregate of each DHIS2 Data Value across *all* CHWs in their area from the *CHW Aggregate* view on the **Targets tab**. By tapping on a target, they can also see each CHW’s contribution towards that total. Supervisors can review and verify with CHWs that everyone has synced and that their data is correct. The Supervisor can communicate with the HRIO when everything has been validated.
 
 ![Supervisors](supervisor.png "Feature Overview Supervisors")
 
 ### HRIO
 
-HRIOs access the CHT App Management tab and select the appropriate DHIS2 Data Set, Organisation Unit, and Period.  They then “Export” a file that is formatted for DHIS2. HRIOs will need access to the “Import/Export” feature in DHIS2 so that they can “Import” the file. Once it has been imported into DHIS2, they review the data from the Data Entry screen in DHIS2.
+HRIOs access the **CHT App Management** tab and select the appropriate *DHIS2 Data Set, Organisation Unit,* and *Period*. They then *Export* a file that is formatted for DHIS2. HRIOs will need access to the **Import/Export** feature in DHIS2 so that they can **Import** the file. Once it has been imported into DHIS2, they review the data from the **Data Entry** screen in DHIS2.
 
 ![Data Entry](data-entry-1.png "Feature Overview Data Entry 1")
 
 
 ## DHIS2 apps
 
-The CHT also includes an [API](https://github.com/medic/cht-core/tree/master/api#get-apiv2exportdhis) that can be called from other applications that returns DHIS2 Data Sets. This means that you can build a [DHIS2 app](https://docs.dhis2.org/master/en/developer/html/apps_creating_apps.html) that pulls data from the CHT and imports it electronically into DHIS2. This would allow Data Entry to control the process directly from DHIS2 without having to access the CHT.
+The CHT also includes an [API](https://github.com/medic/cht-core/tree/master/api#get-apiv2exportdhis) that can be called from other applications that returns DHIS2 Data Sets. This means that you can build a [DHIS2 app](https://docs.dhis2.org/master/en/developer/html/apps_creating_apps.html) that pulls data from the CHT and imports it electronically into DHIS2. This would allow **Data Entry** to control the process directly from DHIS2 without having to access the CHT.
```

Comment:
I think this one is not a UI element, but rather a person, in which case it should not be bold.

Reply:
@abbyad  I thought it might be the Data Entry screen in DHIS2 we mention in the previous section?